TOMP-API.yaml

```diff
@@ -1361,26 +1401,24 @@ components:
         assetClass:
           $ref: "#/components/schemas/assetClass"
 
-    compositeLeg:
+    compositeOption:
       description: this leg type should be used when returning multiple legs to fullfil a single request from A to B. For instance handling overlegs or when acting as broker for multiple sub contractors.
-      allOf:
-        - $ref: "#/components/schemas/planningResult"
-        - type: object
-          properties:
-            id:
-              description: unique ID (TO's perspective) for this option. This ID is used during the complete process booking of a specific asset or an asset of a specific type. If the availability-request is not fired within f.i. 30 minutes, it can savely be removed.
-              type: string
-            pricing:
-              $ref: "#/components/schemas/fare"
-            legs:
-              type: array
-              items:
-                $ref: "#/components/schemas/operatorLeg"
-            conditions:
-              description: references to the 'conditions' array (start of this object).
-              type: array
-              items:
-                type: string
+      type: object
```

Comment:
If we're going to change the leg structure, please view my proposal I sent you by mail.
![image](https://user-images.githubusercontent.com/10400054/84152155-9155eb00-aa64-11ea-8328-dedaa3d0f8bc.png)